src/view/editingLayerInner.tsx

```diff
@@ -81,7 +81,7 @@ export class EditingLayerInnerComp extends Component<Props> {
       this.contentEl.appendChild(editorEl);
       this.editor = cellEditor;
 
-      const editorWidth = (this.editor.el as HTMLElement).getBoundingClientRect().width;
+      const editorWidth = (editorEl as HTMLElement).getBoundingClientRect().width;
```

Comment:
The type assertion also can be removed. Could you remove it?

Reply:
Okay.